index.html

```diff
@@ -307,5 +307,42 @@ <h1>Skills</h1>
         </div>
     </section>
     <!--   End of About Section   -->
+    <footer class="contact">
+        <div class="footer-container">
+            <div class="wrapper">
+                <div class="text-container">
+                    <h1>Contact me</h1>
+                    <p>
+                        If you have an application you are interested in developing,
+                        a feature that you need built or a project that needs coding. I’d love to help with it 
+                    </p>
+                </div>
+    
+                <div class="form-container">
+                    <form action="https://formspree.io/f/mwkzoyge" method="post">
+    
+                       <ul>
+                        <li>
+                            <label for="name"></label>
+                            <input type="text" id="name" name="name" value="Bryte Litty" maxlength="30" required>
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] Kindly replace the value with the placeholder `why?` because the placeholder attribute specifies a short hint that describes the expected value (type of value) of an input field. whilst the `value` You need to erase the value and then add the new value.